HW_1/task_3.py

```diff
@@ -0,0 +1,34 @@
+"""
+Напишите программу, которая принимает на вход координаты точки (X и Y) и выдаёт номер четверти плоскости,
+в которой находится эта точка (или на какой оси она находится).
+
+Ввод: два значения типа <int>
+Вывод: значение типа <int> либо значение типа <str>
+
+Пример:
+
+34
+-30
+4
+
+2
+4
+1
+
+-34
+0
+Точка на отрицательной части оси абсцисс
+"""
+x = int(input('Enter the number X: '))
+y = int(input('Enter the number Y: '))
+
+if x == 0 or y == 0:
+    print('Input Error')
+elif x > 0 and y > 0:
```

Comment:
Условия подобного рода можно свернуть относительно повторяющегося значения
```suggestion
elif x > 0 < y:
```